src/allmydata/test/test_istorageserver.py

```diff
@@ -194,20 +194,6 @@ def abort_or_disconnect_half_way(self, abort_or_disconnect):
         )
         yield allocated[0].callRemote("write", 0, b"2" * 1024)
 
-    def test_disconnection(self):
```

Comment:
Disconnection testing only makes sense for Foolscap, so this code was moved to be in the Foolscap-specific test case class, rather than shared one.